### PR TITLE
Add uniswap worldchain trades

### PR DESCRIPTION
**Pull request by Hosuke**
Add uniswap worldchain trades

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

### CODE REVIEW

File: dbt_subprojects/dex/models/trades/worldchain/_schema.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+version: 2
+
+models:
+  - name: dex_worldchain_base_trades
+    tests:
+      - check_dex_info_relationship
+  
+  - name: uniswap_v3_worldchain_base_trades
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/worldchain/_schema.yml:8]: [uniswap_v3_worldchain_base_trades preview](https://dune.com/queries/4156407?category=decoded_project&namespace=uniswap_v3)

File: dbt_subprojects/dex/seeds/trades/uniswap_worldchain_base_trades_seed.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+worldchain,uniswap,3,2024-10-13,0xf0fb23913be35bde58ff9cd1bde5b4e7e1b89c9fd1ca3e1b68bb3642a3a6826f,15,0x2cfc85d8e48f8eab294be644d9e25c3030863003,0x79a02482a880bce3f13e09da970dc34db4cd24d1,4725968,5115943631049460102,9199756
```
Comment (Hosuke) [inline dbt_subprojects/dex/seeds/trades/uniswap_worldchain_base_trades_seed.csv:2]: https://worldchain-mainnet.explorer.alchemy.com/tx/0xf0fb23913be35bde58ff9cd1bde5b4e7e1b89c9fd1ca3e1b68bb3642a3a6826f

File: dbt_subprojects/dex/seeds/trades/uniswap_worldchain_base_trades_seed.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+worldchain,uniswap,3,2024-10-13,0xb3bcf7ab7b5d55f0bfbac2bf723c8df074fa237a591639748196af02c73c8b31,5,0x79a02482a880bce3f13e09da970dc34db4cd24d1,0x2cfc85d8e48f8eab294be644d9e25c3030863003,4723857,17630073,10000000000000000000
```
Comment (Hosuke) [inline dbt_subprojects/dex/seeds/trades/uniswap_worldchain_base_trades_seed.csv:3]: https://worldchain-mainnet.explorer.alchemy.com/tx/0xb3bcf7ab7b5d55f0bfbac2bf723c8df074fa237a591639748196af02c73c8b31

**Review by jeff-dude — APPROVED** [review]
thank you!

**Pull request merged**